proto/sdv/databroker/v1/types.proto

```diff
@@ -144,56 +144,52 @@ message Datapoint {
 }
 
 message Metadata {
-  int32 id               = 1;
-  EntryType entry_type   = 2;
-  string name            = 4;
-  DataType data_type     = 5;
-  ChangeType change_type = 6;  // CONTINUOUS or STATIC or ON_CHANGE
-  string description     = 7;
-
-  ValueRestriction value_restriction = 10;
-  // int32             min_update_hz       = 10; // Only for CONTINUOUS
-  // int32             max_update_hz       = 11; // Only for CONTINUOUS
+  int32 id                             = 1;
+  EntryType entry_type                 = 2;
+  string name                          = 4;
+  DataType data_type                   = 5;
+  ChangeType change_type               = 6;  // CONTINUOUS or STATIC or ON_CHANGE
+  string description                   = 7;
+
+  // Value restrictions
+  ValueRestrictions value_restrictions = 10;
```

Comment:
Do we know if there are anyone actually using ValueRestriction in this old API?
Because this will be a breaking change for anyone actually using this, right?